semgrep_output_v1.atd

```diff
@@ -511,6 +511,7 @@ type error_type
    * *)
   | PatternParseError0 <json name="Pattern parse error">
   | IncompatibleRule0 <json name="Incompatible rule">
+  | DependencyResolutionError of resolution_error
```

Comment:
wdyt about adding to the documentation to make it clear what version we introduced this error!

Reply:
yes, we usually do that, like (* since 1.94.0 *).

Reply:
updated!